src/main/kotlin/net/insprill/robotinsprill/command/slash/Post.kt

```diff
@@ -0,0 +1,60 @@
+package net.insprill.robotinsprill.command.slash
+
+import dev.kord.common.entity.TextInputStyle
+import dev.kord.core.behavior.interaction.modal
+import dev.kord.core.behavior.interaction.response.respond
+import dev.kord.core.event.interaction.ChatInputCommandInteractionCreateEvent
+import dev.kord.rest.builder.component.ActionRowBuilder
+import dev.kord.rest.builder.interaction.ChatInputCreateBuilder
+import dev.kord.rest.builder.interaction.channel
+import net.insprill.robotinsprill.RobotInsprill
+import net.insprill.robotinsprill.form.FieldSize
+
+class Post(private val robot: RobotInsprill) : SlashCommand() {
+    override val description: String
+        get() = "Post in the current channel"
+
+    override suspend fun execute(context: ChatInputCommandInteractionCreateEvent) {
+        val form =
+            robot.config.forms.firstOrNull() { it.channel == (context.interaction.command.channels["target"] ?: context.interaction.channel).id }
+
+        if (form == null) {
+            context.interaction.deferEphemeralResponse()
+                .respond { content = "There is not forms to fill out here u dummy" }
```

Comment:
Please make this message customizable in the config

Reply:
Added in 4750c50c